onnxruntime/core/graph/function.cc

```diff
@@ -152,6 +152,11 @@ static std::unique_ptr<ONNX_NAMESPACE::OpSchema> CreateSchema(const Graph& graph
   op_schema->SetDomain(meta_def->domain);
   op_schema->SetDoc(meta_def->doc_string);
   op_schema->SinceVersion(meta_def->since_version);
+
+  if (meta_def->type_and_shape_inference_function) {
```

Comment:
Probably need an ifdef around this usage as well

Reply:
Seems to pass the minimal CI without it; I guess `function.cc` and the unit test are never included in the minimal build?